programs/castle-vault/src/state.rs

```diff
@@ -74,9 +74,12 @@ pub struct Vault {
 
     pub config: VaultConfig,
 
+    // Actual allocation retrieved by refresh
+    pub actual_allocations: Allocations,
```

Comment:
I think it's more intuitive to have `expected_allocations` or `target_allocations` vs. `actual_allocations`, because it's what the algorithm wants vs. what's actually achieved.

Reply:
Sure that seems fine, can you make the corresponding change for `allocations` to `target_allocations` in that case?